File: (1IEE14) Lab 1/calcula_final.py

import sys

def nota_del_final(a,b):
    res=(110 - 3*a - 3*b)/4
    print("Calculando por cuánto me voy: ",res)
    return res

if len(sys.argv) != 3:
    #En caso de error por que se han ingresado más de dos argumentos
    print("Debe ingresar exactamente 2 notas")

x = int(sys.argv[1])
y = int(sys.argv[2])

resultado = nota_del_final(x,y)
print(resultado)

File: (1IEE14) Lab 1/calcula_final.c

#include <stdio.h>
#include <stdlib.h>

int nota_del_final(int a,int b){
    return (110-3*a-3*b)/4;
}

int main(int argc, char *argv[]){
    if(argc != 3){
        printf("Debe ingresar exactamente 2 notas\r\n");
        return 0;
    }
    int x = atoi(argv[1]);
    int y = atoi(argv[2]);

    printf("Me voy por: %d",nota_del_final(x,y));
    //return 0;
}
